exchanges/bitfinex/bitfinex_wrapper.go

```diff
@@ -649,13 +649,23 @@ func (b *Bitfinex) SubmitOrder(ctx context.Context, o *order.Submit) (*order.Sub
 	var orderID string
 	status := order.New
 	if b.Websocket.CanUseAuthenticatedWebsocketForWrapper() {
-		orderID, err = b.WsNewOrder(&WsNewOrderRequest{
+		symbolPrefix := "t"
+		if o.AssetType == asset.MarginFunding {
+			symbolPrefix = "f"
+		}
+		req := &WsNewOrderRequest{
 			CustomID: b.Websocket.AuthConn.GenerateMessageID(false),
-			Type:     o.Type.String(),
-			Symbol:   fPair.String(),
-			Amount:   o.Amount,
-			Price:    o.Price,
-		})
+			// spot needs to use the EXCHANGE variants for types
+			Type:   "EXCHANGE " + o.Type.String(),
```

Comment:
Should this not be like the below scenario?
```go
	if o.AssetType == asset.Spot {
			orderType = "EXCHANGE " + orderType
		}
```

Reply:
Good catch. I had a brain fart that the if condition above was for spot. It's not.

Reply:
👍